pypesto/ensemble/ensemble.py

```diff
@@ -1222,13 +1222,22 @@ def calculate_cutoff(
     percentile: float = 95,
     cr_option: str = SIMULTANEOUS,
 ):
-    """
-    Calculate the cutoff of the ensemble.
+    r"""
+    Calculate the cutoff of the objective function values of the ensemble.
 
     Based on the number of parameters of the problem. Based on the
     assumption that the difference of the nllh's of the true and optimal
     parameter is chi^2 distributed with n_theta degrees of freedom.
 
+    The ensemble is created based on
+    :math: `-2\log(\mathcal{L}(\theta)/\mathcal{L}(\hat{\theta})) =
+    -2\log(\mathcal{L}(\theta)) - (-2\log(\mathcal{L}(\hat{\theta}))) =
+    2(J(\theta) - J(\hat{\theta}))) \leq \Delta_{\alpha}`, where :math: `mathcal{L}` is the likelihood,
+    :math: `J` is the negative log-likelihood,:math: `\Delta_{\alpha}` is a percentile of the
+    :math: `\chi^2` distribution and :math: `J(\hat{\theta})` is the smallest objective function value
+    found during optimization. The ensemble contains all the parameter vectors :math: `\theta` that satisfy
+    :math: `J(\theta)\leq J(\hat{\theta}) + \Delta_{\alpha}/2`.
```

Comment:
This doesn't appear to render correctly
https://pypesto--1566.org.readthedocs.build/en/1566/api/pypesto.ensemble.html#pypesto.ensemble.calculate_cutoff

There is an example here, might just need some newlines
https://amici.readthedocs.io/en/latest/_modules/amici/import_utils.html#noise_distribution_to_cost_function